cfgov/unprocessed/css/organisms/image-text-50-50-group.less

```diff
@@ -16,13 +16,13 @@
     - cfgov-molecules
 */
 .o-image-text-50-50-group {
-    // TODO: Find another way to handle spacing
+    // TODO: Find another way to handle spacing here and in link-blob-group
```

Comment:
I've got an idea for this I'm testing on the homepage. Now that it's merged, I'm going to extend the info-unit stuff here.